### PR TITLE
Comment button removed

### PR DESCRIPTION
**Pull request by viktor992**
Add comment button has been removed. The comment input box is shown
instead.

See #63

### CODE REVIEW

**Comment by mitar** [comment]
I think I would propose that we make everything the same as when displaying an already existing comment. So when user makes a selection, we add a dummy comment entry on the side, if necessary move existing displayed comments apart. Form looks like a rendered comment, just that it has a form. But it should be of same width and positioned along other comments. Oh, and because cursor is inside the selection, that comment with a form is focused. Until the user clicks a button nothing is stored on the server, but we could have some client-side only representation of that comment-to-be to make logic reusable between displaying this "new form" comment and regular comment.

**Comment by viktor992** [comment]
Ready for review.

**Comment by mitar** [comment]
For "Add highlights before comments " commit, can you add a comment before `emit` why is it there? So in general, when you fix a bug, also add a comment. Because it obviously is not intuitive why it has to be there and there was a bug. So we should explain what is being fixed, linking to GitHub issue, so that people know why is that there.

**Comment by viktor992** [comment]
Added code comments about bugfix #69.

File: client/components/sidebar.vue [hunk 1]
```diff
@@ -1,64 +1,67 @@
 <template>
-  <v-container fluid class="sidebar__users">
+  <v-container @mousedown.stop fluid class="sidebar__users">
```
Comment (mitar) [inline client/components/sidebar.vue]: From my experience preventing propagation of events often backfires. For example, many times when you have some overlay (like dialog) you want it to be closed if you click outside of it. But if user clicks on element which is stopping propagation then this does not happen.

So here you are preventing propagation for whole sidebar. Is this really what you want?

And elsewhere in the app there is also `.stop` around. Is it really necessary? Isn't preventing default action often enough?

**Pull request merged**